databases/week1/HW.sql

```diff
@@ -0,0 +1,54 @@
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM hyf_class.task;
```

Comment:
This is correct and works, but i wound recommend not specifying the database name in the queries.
If you do `USE hyf_class;` you can just run the query as `SELECT COUNT(*) FROM task;`. That way the query works even if the database name is different on another persons computer.

Reply:
Hi Flemming (@flemeur),
Thanks a million for your time reviewing my HW, although I have sent it very late. And Thousands of million tanks for your comments and help. Great hints. I tried to remove DB name from Table name and use lowercase letters for aliases.
Thank you very much.